pinot-common/src/main/java/org/apache/pinot/pql/parsers/Pql2Compiler.java

```diff
@@ -57,10 +61,16 @@
 @ThreadSafe
 public class Pql2Compiler implements AbstractCompiler {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(Pql2Compiler.class);
+
+  private static final boolean VALIDATE_CONVERTER =
```

Comment:
Any reason to choose env var over config?

Reply:
will do that in subsequent PR's.